api/util/importer.js

```diff
@@ -4,23 +4,37 @@ var locations = rfr('lambda/locations');
 var resources = rfr('lambda/resources');
 var bookings = rfr('lambda/bookings');
 var cognito = rfr('util/cognito');
-var logger = rfr('util/logger');
+
+
+var SampleGroups = [
+  {
+    name: 'adminGroup',
+    description: 'Cognito user group for administrators',
+    precedence: 0,
+    iam: 'something'
+  },
+  {
+    name: 'clientGroup',
+    description: 'Cognito user group for spacefinder users',
+    precedence: 0,
```

Comment:
yes, changed it to 1. Technically, you can have two [groups] in the user pool with the same precedence. But I agree with you that having different values here makes it more readable.


[groups]: http://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-pools-user-groups.html#assigning-precedence-values-to-groups